Automattic/jetpack
# Admin: Show survey notice when the new stats UI is disabled

Fixes #28728.

## Proposed changes:

<img width="1067" alt="image" src="https://user-images.githubusercontent.com/4044428/217411395-c31c10c3-c833-4c95-b1c8-b40494291f2f.png">

Adds a new notice above the Odyssey Stats toggle that's only shown when Odyssey Stats is disabled. The survey link uses Jetpack redirects (PCYsg-pY7-p2).

This PR also moves the Jetpack Stats settings card above the Google Analytics settings card.

### Other information:


## Jetpack product discussion


p1HpG7-khi-p2#comment-60393

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Apply this change to your WordPress site with the Jetpack plugin.
* Navigate to Jetpack -> Settings -> Traffic via the wp-admin sidebar.
* Ensure you see the survey notice if the new stats experience is disabled.
* Ensure you do not see the notice when the new stats experience is enabled.
